giskard/core/suite.py

```diff
@@ -219,6 +220,16 @@ def _to_dto(self, label: Optional[str], client: GiskardClient, project_key: str)
             completionDate=self.completion_date.isoformat(),
         )
 
+    def to_json(self, filename=None):
+        results = {}
+        for suite_result in self.results:
+            results[suite_result.test_name] = "Passed" if suite_result.result.passed else "Failed"
```

Comment:
can we add all the meta data that is available / could be useful? 
- metric value
- model name
- dataset(s) name(s)